src/libpmemset/pmemset_utils.c

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright 2020, Intel Corporation */
+
+/*
+ * pmemset_utils.c -- libpmemset utilities functions
+ */
+
+#include <errno.h>
+#include "alloc.h"
+#include "libpmemset.h"
+#include "out.h"
+#include "pmemset_utils.h"
+#include "util.h"
+
+/*
+ * pmemset_malloc -- allocate a buffer and handle an error
+ */
+void *
+pmemset_malloc(size_t size, int *err)
```

Comment:
Do we need this function (pmemset_malloc)? The same is already in pmem2 (pmem2_malloc).

Reply:
It's actually from my commit.
Turned out it's needed to keep the libpmem2 internal APIs out of libpmemset.